src/frequenz/sdk/timeseries/_ringbuffer/buffer.py

```diff
@@ -278,39 +279,41 @@ def is_missing(self, timestamp: datetime) -> bool:
         return any(map(lambda gap: gap.contains(timestamp), self._gaps))
 
     def _update_gaps(
-        self, timestamp: datetime, newest: datetime, new_missing: bool
+        self, timestamp: datetime, newest: datetime, should_be_in_gaps: bool
     ) -> None:
         """Update gap list with new timestamp.
 
         Args:
             timestamp: Timestamp of the new value.
             newest: Timestamp of the newest value before the current update.
-            new_missing: if true, the given timestamp will be recorded as missing.
-
+            should_be_in_gaps: if true, the given timestamp will be recorded as missing.
```

Comment:
Silly but just for consistency:

```suggestion
            should_be_in_gaps: If `True`, the given timestamp will be recorded as missing.
```

Also from the docs it looks like maybe `record_as_missing` might be even more clear, I find `should_be_in_gaps` not super clear TBH.